cli/lib/localization.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const path = require('path');
+const toml = require('@iarna/toml');
+const osLocale = require('os-locale');
+
+module.exports = translate;
+
+let translations = null;
+let locale = null;
+
+const replace = (str, map) =>
```

Comment:
I love function expressions in consts, but the style in this project seems to be function declarations, should  this one be changed?